2012/06/12 03:37:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\xampp\php\library\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\php\library\framework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 C:\xampp\php\library\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\svn\index.php(16): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/12 03:52:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\xampp\php\library\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\php\library\framework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 C:\xampp\php\library\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\svn\index.php(16): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/12 04:04:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\xampp\php\library\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\php\library\framework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 C:\xampp\php\library\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\svn\index.php(16): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/12 04:14:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "public/userfiles/image/020512/image/demo.png".' in C:\xampp\php\library\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\php\library\framework\web\CWebApplication.php(135): CWebApplication->runController('public/userfile...')
#1 C:\xampp\php\library\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\svn\index.php(16): CApplication->run()
#3 {main}
REQUEST_URI=/public/userfiles/image/020512/image/demo.png
HTTP_REFERER=http://dos.vn/
---
2012/06/12 04:14:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "public/userfiles/image/230412/image/demo.png".' in C:\xampp\php\library\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\php\library\framework\web\CWebApplication.php(135): CWebApplication->runController('public/userfile...')
#1 C:\xampp\php\library\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\svn\index.php(16): CApplication->run()
#3 {main}
REQUEST_URI=/public/userfiles/image/230412/image/demo.png
HTTP_REFERER=http://dos.vn/
---
